### PR TITLE
Madlib 504

### PR DESCRIPTION
**Pull request by renyi533**
Fix for madlib 504

### CODE REVIEW

**Comment by haradh1** [comment]
As you promised in earlier PRs, please clear your branch and send only commits that actually affect.  The merge and review work is painful.
Please do this clearance for _all_ of existing your (I mean china team's) PRs as soon as possible.

**Comment by renyi533** [comment]
Hi Hitoshi,

I also want to clear the branches but failed. I tried to update the commit range but it did not work well.

Can you give me some guidance on that?

Thanks,
Ren Yi

**Comment by haradh1** [comment]
Please refer to git document for the basic usage of git.

My recommendation is:
- sync to master
- create a branch from it
- apply the change you want with cherry-pick or merge

Anyway this is so easy and there are couple of ways.

**Comment by renyi533** [comment]
I already synced to master.

Please see the commit of "8e8f7fc       Merge remote-tracking branch 'upstream/master' into madlib_504".

Why the previous commits are still present in this pull request?

**Comment by haradh1** [comment]
No, if you cloned freshly and applied only the commit you need, there shouldn't be extraneous commits shown here.  Please do git reset --hard upstream/master, or discard your repository and clone it again.

File: methods/cart/src/pg_gp/dt.sql_in [hunk 1]
```diff
@@ -2323,7 +2339,8 @@ m4_changequote(>>>`<<<, >>>'<<<)
                training_table_meta,
                sp_crit,
                grow_tree,
-               'find_best_answer_table'
+               'find_best_answer_table',
+			   h2hmv_routine_id
```
Comment (haradh1) [inline methods/cart/src/pg_gp/dt.sql_in]: It looks like indentation is inconsistent.

Comment (renyi533) [inline methods/cart/src/pg_gp/dt.sql_in]: I will change and resubmit it.

**Comment by haradh1** [comment]
Pulled with a little modification by me.

**Pull request closed without merging**